### ISSUE TITLE
Container images from jsonnet configuration do not match the version tag

### ISSUE THREAD

**Issue by jphx** [grafana/loki#7719]
If I configure Loki using Tanka/jsonnet from the `v2.7.0` tag, it still uses `2.6.1` images.  This is the second release in a row where I've noticed the image tags are incorrect.   When using the `v2.6.1` tag, the images it calls for are `2.6.0`.  Even in `main`, I see the older image tags [are still used](https://github.com/grafana/loki/blob/main/production/ksonnet/loki/images.libsonnet#L7).  It would be easier to use the Tanka/jsonnet configuration if the tagged commit had the matching image tags.

**Comment by jphx** [comment]
This is wrong again in the v2.7.1 jsonnet, which calls for the v2.7.0 container images.

**Comment by jphx** [comment]
This is wrong again in the v2.7.2 jsonnet, which still calls for the v2.7.0 container images.

**Comment by jphx** [comment]
This is wrong again in the v2.7.3 jsonnet, which calls for the v2.7.2 container images.

**Comment by JStickler** [comment]
Closing as stale. This issue was opened against a version of Loki that is no longer supported. If this issues still exists in more recent versions of Loki, please leave a comment and we can reopen the issue.
